source/opt/module.h

```diff
@@ -66,8 +66,10 @@ class Module {
   inline void AddEntryPoint(std::unique_ptr<Instruction> e);
   // Appends an execution mode instruction to this module.
   inline void AddExecutionMode(std::unique_ptr<Instruction> e);
-  // Appends a debug instruction (excluding OpLine & OpNoLine) to this module.
-  inline void AddDebugInst(std::unique_ptr<Instruction> d);
+  // Appends a debug 1 instruction (excluding OpLine & OpNoLine) to this module.
```

Comment:
Added. Should I add that comment to all function prototypes containing debug1 or debug2, like `debug1_clear()`?